src/SharpLearning.Common.Interfaces/IPredictor.cs

```diff
@@ -13,5 +15,7 @@ public interface IPredictor<TPrediction>
         /// <param name="observation"></param>
         /// <returns></returns>
         TPrediction Predict(double[] observation);
+
+        TPrediction[] Predict(F64Matrix observations);
```

Comment:
@andjor The new Predict method overload is missing the XML comment. This causes a compiler warning since, so please add an XML comment like on the original method.

You are also welcome to add an actual description to the two methods.

For the single observation case: `Predicts a single observation`
For the multiple observations case: `Predicts a set of observations`